upcloud/resource_upcloud_floating_ip_address_test.go

```diff
@@ -189,11 +189,9 @@ func testUpcloudFloatingIPAddressCreateWithServerConfig(serverNames []string, as
   			hostname = "mydebian.example.com"
   			plan     = "1xCPU-2GB"
 
-  			storage_devices {
-    			action = "create"
-    			size   = 10
-    			tier   = "maxiops"
-  			}
+        template {
+          storage = "01000000-0000-4000-8000-000020050100"
```

Comment:
Not that big of a problem, but there's different indentation here and in some other places where this PR has touched.